libsolidity/codegen/ir/IRGenerationContext.cpp

```diff
@@ -224,7 +224,7 @@ void IRGenerationContext::internalFunctionCalledThroughDispatch(YulArity const&
 
 YulUtilFunctions IRGenerationContext::utils()
 {
-	return YulUtilFunctions(m_evmVersion, m_revertStrings, m_functions);
+	return YulUtilFunctions(m_evmVersion, std::nullopt, m_revertStrings, m_functions);
```

Comment:
This is in IR codegen so EOF version should be passed in:
```suggestion
	return YulUtilFunctions(m_evmVersion, m_eofVersion, m_revertStrings, m_functions);
```